lib/bloc/invoice/invoice_bloc.dart

```diff
@@ -156,6 +156,9 @@ class InvoiceBloc with AsyncActionsHandler {
         .map((s) {
           String lower = s.toLowerCase();
           if (lower.startsWith("lightning:")) {
```

Comment:
Checking for lower case repeats itself in several places. The reason is I guess to to test for prefix 'lightning:'.
Can we use parseLlightningAddress instead? Maybe with a boolean parameter 'prefixed' that enforce prefix or not?
It is better to encapsulate this condition.

Reply:
Will do.

Reply:
> Checking for lower case repeats itself in several places. The reason is I guess to to test for prefix 'lightning:'. [...]
> It is better to encapsulate this condition.

To clarify: Do you want me to change all the places that parse/test for a `lightning:` URIs to use a single utility function, even for this case when we are testing invoices, not lightning-addresses?

Reply:
That is, if you'd like me to refactor all the places that check the `lightning:` prefix, even when they're not checking for lightning-addresses, is it okay if I do that in a different PR?

Reply:
I think now that you added the isLightningURI you already encapsulated the logic so no need to do more than that now.

Reply:
Okay. Thanks.